File: despatch.Domain/Models/Default/Response/MessageResponse.cs
using System;

namespace despatch.Domain.Models.Default.Response
{
    public class MessageResponse : BaseEntityResponse
    {
        public string Text { get; set; }
        public DateTime SendDate { get; set; }

        public string UserId { get; set; }
        public string RoomId { get; set; }

        public virtual UserResponse User { get; set; }
        public virtual RoomResponse Room { get; set; }
    }
}

File: despatch.WEB/app/src/swagger/models/message-response.ts
/* tslint:disable */
import { BaseEntityResponse } from './base-entity-response';
import { RoomResponse } from './room-response';
import { UserResponse } from './user-response';
export interface MessageResponse extends BaseEntityResponse{
  room?: RoomResponse;
  roomId?: string;
  sendDate: string;
  text?: string;
  user?: UserResponse;
  userId?: string;
}
